Thorium-Sim/thorium-nova
# First cards

## Description

Adds a simple login card.

## Related Issue








Refs #132 

## How do you know the changes work correctly?






Manual Test
- Make sure you have the latest default plugin installed (see the readme)
- Start a flight
- Assign your client to a station
- You should see the login screen.

## Screenshots (if appropriate):

## Types of changes



      change)

## Checklist:
